packages/bottom-tabs/src/views/BottomTabView.tsx

```diff
@@ -123,7 +124,7 @@ export default function BottomTabView(props: Props) {
             <MaybeScreen
               key={route.key}
               style={[StyleSheet.absoluteFill, { zIndex: isFocused ? 0 : -1 }]}
-              visible={isFocused}
+              visible={isFocused || !!keepInactiveScreensVisible}
```

Comment:
This needs to be a separate prop on `MaybeScreen` (e.g. `enabled`). The `visible` prop also sets other props such as disabling `pointerEvents` which wouldn't be appropriate to keep enabled for unfocused screens.